Lesson3_HW/Program.cs

```diff
@@ -1,33 +1,121 @@
 ﻿using System;
+using System.Runtime.Intrinsics.X86;
+using System.Text.RegularExpressions;
 
 public enum Gender
 {
     Male,
     Female,
-    Unknown
+    Mechanic
 }
 
 class Program
 {
     private static void Main(string[] args)
     {
-        Console.WriteLine("Empty project");
+        User user1 = new User(19);
+
+
+        while (true)
+        {
+            try
+            {
+                Console.WriteLine("Enter your gender Male (or 0), Female (or 1), or Mechanic (or 2). :");
+                user1.Gender = Console.ReadLine();
+                break;
+            }
+            catch (ArgumentException ex)
+            {
+                Console.WriteLine(ex.Message);
+            }
+        }
+
+        while (true)
```

Comment:
It's better avoid infinite loop, perhaps better to do it in do{}while{} loop with some condition to exit